src/mlpack/methods/grad_boosting/grad_boosting.hpp

```diff
@@ -0,0 +1,220 @@
+/**
+ * @file methods/grad_boosting/grad_boosting.hpp
+ * @author Abhimanyu Dayal
+ *
+ * Gradient Boosting class. Gradient Boosting uses weak learners (primarily 
+ * decision stumps), and trains them sequentially, such that future learners are 
+ * trained to detect the errors (or gradients) of previous learners. 
+ * The results obtained from all of these learners are subsequently 
+ * aggregated to give a final result.
+ * 
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+// Include guard - Used to prevent including double copies of the same code
+#ifndef MLPACK_METHODS_GRADBOOSTING_GRADBOOSTING_HPP
+#define MLPACK_METHODS_GRADBOOSTING_GRADBOOSTING_HPP
+
+// Importing base components required to write mlpack methods.
+#include <mlpack/core.hpp>
+
+// Only using decision trees for now, therefore only including decision tree functionalities.
+// #include <mlpack/methods/decision_tree/decision_tree.hpp>
```

Comment:
This should work, and would be the preferred way in including the decision tree header.

Reply:
Implemented this :+1: